src/EnergyPlus/PluginManager.cc

```diff
@@ -496,6 +496,36 @@ PluginManager::PluginManager(EnergyPlusData &state)
                 fs::path sanitizedInputFileDir = PluginManager::sanitizedPath(state.dataStrGlobals->inputDirPath);
                 PluginManager::addToPythonPath(state, sanitizedInputFileDir, false);
             }
+
+            std::string epInDirFlagUC = "YES";
+            try {
+                epInDirFlagUC =
+                    EnergyPlus::UtilityRoutines::MakeUPPERCase(fields.at("add_epin_environment_variable_to_search_path").get<std::string>());
+            } catch (nlohmann::json::out_of_range &e) {
+                // defaulted to YES
+            }
+            if (epInDirFlagUC == "YES") {
+                std::string epin_path;
+                get_environment_variable("epin", epin_path);
+                fs::path epinPathObject = fs::path(epin_path);
+                if (epinPathObject.empty()) {
+                    EnergyPlus::ShowWarningMessage(
+                        state,
+                        "PluginManager: Search path inputs requested adding epin variable to Python path, but epin variable was empty, skipping."
+                    );
+                } else {
+                    if (FileSystem::pathExists(epinPathObject)) {
+                        fs::path sanitizedEnvInputDir = PluginManager::sanitizedPath(epinPathObject);
+                        PluginManager::addToPythonPath(state, sanitizedEnvInputDir, true);
+                    } else {
+                        EnergyPlus::ShowWarningMessage(
+                            state,
+                            "PluginManager: Search path inputs requested adding epin variable to Python path, but epin variable value is not a valid existent path, skipping."
+                        );
+                    }
+                }
+            }
```

Comment:
- Assume the input field value is true, then read the value from the IDF
- If it is true, then try to read the environment variable value
- If the environment variable doesn't exist, issue a warning for that odd combination
- Otherwise read the environment variable value, and if the path exists, add it to the Python search path.
  - I thought about this one for a bit.  Should we concern ourselves with checking if the path exists on the drive?  I mean maybe, maybe not, but in this case, the Python interpreter is literally milliseconds away from actually using that path to find Python stuff, so I leaned on providing a nice warning if the directory doesn't exist.